src/App.tsx

```diff
@@ -1,9 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+type State = string;
+
+type Movies = {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string
+};
+
+export function getFilteredMovies(movies: Movies[], filter: string) {
+  const visibleMovies = [...movies];
+
+  return visibleMovies.filter(movie => movie.title
+    .toLowerCase()
+    .includes(filter.toLowerCase().trim())
+    || movie.description.toLowerCase().includes(filter.toLowerCase().trim()));
+}
+
 export const App: React.FC = () => {
+  const [movieListState, setMovieListState] = useState<State>('');
```

Comment:
`movieListState` isn't a meaningful name. It's a filter. So name like "filter", "query" or something